resources/lang/de/validation.php

```diff
@@ -9,37 +9,7 @@
     'alpha_dash' => ':attribute darf nur aus Buchstaben, Zahlen, Binde- und Unterstrichen bestehen.',
     'alpha_num' => ':attribute darf nur aus Buchstaben und Zahlen bestehen.',
     'array' => ':attribute muss ein Array sein.',
-    'attributes' => [
```

Comment:
I meant to comment here earlier:
Why are you removing these? How will we migrate these into the database?

Reply:
These are downloaded from the web, so these are not synced with our en/hu translations

Reply:
Okay

Reply:
But isn't it true for the Hungarian translations?